mix.exs

```diff
@@ -36,10 +36,10 @@ defmodule ExAws.Mixfile do
       {:xml_builder, "~> 0.0.6", optional: true},
       {:sweet_xml, "~> 0.6", optional: true},
       {:ex_doc, "~> 0.14", only: :dev},
-      {:hackney, "1.6.3 or 1.6.5 or 1.7.1 or 1.8.6", optional: true},
-      {:poison, ">= 1.2.0", optional: true},
+      {:hackney, "1.8.6", optional: true},
+      {:poison, "~> 3.1.0", optional: true},
```

Comment:
> Poison doesn't maintain a changelog so it's a bit hard to know what actually changes between versions, so I don't want to for people to upgrade this either.

I didn't know such background of Poison :( ...
Thanks for the information, now I think it doesn't need for update :)